UI/window-basic-main-outputs.cpp

```diff
@@ -1977,6 +1983,48 @@ bool AdvancedOutput::StartReplayBuffer()
 	return true;
 }
 
+void AdvancedOutput::SaveReplayBuffer()
+{
+	if (!ffmpegOutput || ffmpegRecording) {
```

Comment:
Since you have no `else` section, instead of having a huge section of indented code, you could swap the conditions and return early.

```
	if (ffmpegOutput && !ffmpegRecording)
		return;
```

Then you continue with the rest of the code with one less indentation level.